docs/courses/csintro1/tilemap/extensions.md

```diff
@@ -0,0 +1,104 @@
+# Activity: Level Design and the Corgio Extension
+
+The design of the area the player is in can make or break a game: if it is too sparse, the game can become boring and uninteresting, whereas if it is overly filled in, the game can screech to an unbearable halt. In this activity, students will use an extension designed to make the development of 'platformer' style games easy, so that they are able to focus on the design and structure of their own levels without getting bogged down in make the gameplay fun.
+
+In this activity, students will:
+* Use a new extension
+* Design their own levels using a ``||scene:tilemap||``
+* Implement a platformer game
+
+## Using the corgio package
+
+To simplify the design process of the game, we will be using the corgio extension. This extension allows you to create a sprite who can jump, run, and fall with only a few blocks in the ``||loops:on start||``; many of these behaviors will continue to run in the background without further input.
+
+To add the corgio extension to your project, open the extensions tab and search for "corgio" - it should be the first result. Alternatively, you can search using a direct link to the package, as follows: [github.com/jwunderl/pxt-corgio](github.com/jwunderl/pxt-corgio).
```

Comment:
maybe this shouldn't be a link, just in backticks? because the link won't work on the doc site without `https://` in front and they are supposed to copy&paste it anyways